wd_helper.au3

```diff
@@ -1395,13 +1396,12 @@ Func _WD_UpdateDriver($sBrowser, $sInstallDir = Default, $bFlag64 = Default, $bF
 						__WD_CloseDriver($sDriverEXE)
 						FileDelete($sInstallDir & $sDriverEXE)
 
+						#Region - Extract new instance of webdriver
 						; Handle COM Errors
 						Local $oErr = ObjEvent("AutoIt.Error", __WD_ErrHnd)
 						#forceref $oErr
-
-						; Extract new instance of webdriver
 						$oShell = ObjCreate("Shell.Application")
-						If @error Then
+						If @error Or FileGetSize($sTempFile) = 0 Or IsObj($oShell.NameSpace($sTempFile)) = 0 Then
```

Comment:
I assume that you've tested this to make sure it solves the issue. Should this section of code be "flagged" for future review if a newer version of Autoit is ever released?

Reply:
> I assume that you've tested this to make sure it solves the issue.

Yes.
I did the test in the same testing scenario as in mentioned issue: https://github.com/Danp2/au3WebDriver/issues/203

> Should this section of code be "flagged" for future review if a newer version of Autoit is ever released?
No. At least as far as you wnated to support `au3WebDriver` on current AutoIt release.
The future release will fix some issues but not all.

btw.
There is still possibility to have some AV interactions.
We can move `IsObj()` checking under `Local $oNameSpace = $oShell.NameSpace($sTempFile)` but it will end in the next indentantion `If ... Then... Else .... EndIf`

Reply:
> Should this section of code be "flagged" for future review if a newer version of Autoit is ever released?

Current BETA and ALPHA works withou the fix in `1404 line`